plugins/woocommerce-blocks/assets/js/atomic/blocks/product-elements/sku/style.scss

```diff
@@ -3,7 +3,16 @@
 	@include font-size(small);
 	overflow-wrap: break-word;
 
-	strong {
-		text-transform: uppercase;
+	font-size: var(--wp--preset--font-size--small);
```

Comment:
With the current `wp-block-post-terms` class this line is no longer needed as theme takes care of the size. 🤔